cli/providers/aws/translate.go

```diff
@@ -75,29 +75,30 @@ func Translate(s *service.Service, e *environment.Environment, config *aws.Confi
 func loadStateful(t *gocf.Template, s *service.Service, e *environment.Environment, ami, vpc, encodedCompose string) error {
 	rand.Seed(time.Now().Unix())
 	subnet := *e.Provider.Subnets[rand.Intn(len(e.Provider.Subnets))]
-	instanceIngressRules := gocf.EC2SecurityGroupRuleList{}
-	for _, port := range s.GetPorts() {
-		portNumber, _ := strconv.ParseInt(port.Port, 10, 64)
-		instanceIngressRules = append(
-			instanceIngressRules,
-			gocf.EC2SecurityGroupRule{
-				CidrIp:     gocf.String("0.0.0.0/0"),
-				IpProtocol: gocf.String("tcp"),
-				FromPort:   gocf.Integer(portNumber),
-				ToPort:     gocf.Integer(portNumber),
-			})
-	}
-
 	securityGroups := &gocf.StringListExpr{
 		Literal: []*gocf.StringExpr{gocf.String(e.Provider.SecurityGroup)}}
-	if len(instanceIngressRules) > 0 {
-		securityGroup := &gocf.EC2SecurityGroup{
-			GroupDescription:     gocf.String(fmt.Sprintf("Security Group for %s", s.GetFullName(e, "-"))),
-			SecurityGroupIngress: &instanceIngressRules,
-			VpcId:                gocf.String(vpc),
+	if s.Public {
```

Comment:
maybe we should extract the SG creation code to its own function?

Reply:
ok